slowmovie.py

```diff
@@ -50,11 +50,11 @@ def generate_frame(in_filename, out_filename, time):
         .run(capture_stdout=True, capture_stderr=True)
     )
 
-def check_mp4(value):
+def check_vid(value):
     if not os.path.isfile(value):
         raise configargparse.ArgumentTypeError("File '%s' does not exist" % value)
     if not supported_filetype(value):
-        raise configargparse.ArgumentTypeError("'%s' is not a supported file type" % value)
+        raise argparse.ArgumentTypeError(f"File '{file}' should be a file with one of the following supported extensions: {', '.join(fileTypes)}")
```

Comment:
`configargparse.ArgumentTypeError` and `argparse.ArgumentTypeError` are basically the same error by design. We're using one in line 55 and a different one in 57. Should we pick one to be consistent?

Reply:
Oops, yes! Copy-paste error. Let's stick with configparse as that's what's imported. Fixed in bb4db81